README.rst

```diff
@@ -32,20 +80,23 @@ When you're going to release a new version of the stack, you should:
 
   Example::
 
-    git tag -f 1.1
-    git tag 1.1-20160731
+    git tag -f 1.1-py3
+    git tag 1.1-py3-20160804
+    git tag 1.1-py3-latest
 
 3. Push the changes and the tags to the repo::
 
-    git push -f origin master <tag1> <tag2>
+    git push -f origin master <tag1> <tag2> <tag3>
 
   Example::
 
-    git push -f origin master 1.1 1.1-20160731
+    git push -f origin master 1.1-py3 1.1-py3-20160804 1.1-py3-latest
```

Comment:
1.1-py3-latest is built automatically from branch-1.1-py3, no need to define a tag and push it